tools/patches/006-bootswatch-slate.patch

```diff
@@ -337,24 +337,24 @@ index 8a98f05..78b141a 100644
      text-decoration: underline;
    }
 
-@@ -514,18 +539,18 @@ legend {
+@@ -539,18 +539,18 @@ legend {
    &-success,
    &-warning,
    &-info {
 -    color: $white;
 +    color: $contrast-bg;
    }
  }
-
+ 
  // Containers ==================================================================
-
+ 
  .jumbotron {
 -  border: 1px solid rgba(0, 0, 0, .6);
 +  border: 1px solid rgba($contrast-fg, .6);
  }
-
+ 
  .list-group {
-   &-item:hover {
+   &-item-action:hover {
```

Comment:
This is new in `4.6.0`?

Reply:
https://github.com/thomaspark/bootswatch/compare/v4.5.3...v4.6.0#diff-4fd0a22ae58c02bd0d4ad4ed0e7de80d4dd8e0a54235cc87e4ce0e2bedb06846R528